README.md

```diff
@@ -427,10 +427,22 @@ acf_accordion([
 ]),
 ```
 
-**Clone** - The [clone field](https://www.advancedcustomfields.com/resources/clone) allows you to select and display existing fields.
+**Clone** - The [clone field](https://www.advancedcustomfields.com/resources/clone) allows you to select and display existing fields or groups.
 
 ```php
-// TODO: Add acf_clone function example.
+acf_text([
+	'label' => 'name',
+	'name' => 'Name',
```

Comment:
@edvwib did you try adding the `key` field here? This should work:

https://github.com/wordplate/acf/blob/ec5d5c2078ab2913edca1e43b249ffa3ab4eeb29/tests/FieldTest.php#L156-L181

Reply:
I got it to work, was using an outdated version of wordplate/acf (5.1) :man_facepalming: